common/nexus/failure.go

```diff
@@ -181,7 +181,7 @@ func ConvertGRPCError(err error, exposeDetails bool) error {
 		if !exposeDetails {
 			errMessage = "request timeout"
 		}
-		return nexus.HandlerErrorf(nexus.HandlerErrorTypeDownstreamTimeout, errMessage)
+		return nexus.HandlerErrorf(nexus.HandlerErrorTypeUpstreamTimeout, errMessage) //nolint:govet
```

Comment:
Why is the linter complaining?

Reply:
printf: non-constant format string in call to github.com/nexus-rpc/sdk-go/nexus.HandlerErrorf

Reply:
Perhaps an alternate `nexus.HandlerError` method (no formatting) would be a nice addition